src/geometry/ArborX_Ray.hpp

```diff
@@ -78,15 +78,16 @@ auto returnCentroid(Ray const &ray) { return ray.origin(); }
 // algorithm checks the intersections both in front and behind the ray.
 //
 // There are few issues here. First, when a ray direction is aligned with one
-// of the axis, a division by zero will occur. This is fine, as usually it
-// results in +inf or -inf, which are treated correctly. However, it also leads
-// to the second situation, when it is 0/0 which occurs when the ray's origin
-// in that dimension is on the same plane as one of the corners of the box
-// (i.e., if inv_ray_dir[d] == 0 && (min_corner[d] == origin[d] || max_corner[d]
-// == origin[d])). This leads to NaN, which are not treated correctly (unless,
-// as in [1], the underlying min/max functions are able to ignore them). The
-// issue is discussed in more details in [2] and the website (key word: A
-// minimal ray-tracer: rendering simple shapes).
```

Comment:
Any strong reason for removing the `key word` comment?

Reply:
Which website are we talking about here? Is that supposed to be a google search?

Reply:
I assume this one: https://www.scratchapixel.com/lessons/3d-basic-rendering/minimal-ray-tracer-rendering-simple-shapes/ray-box-intersection.html

Reply:
This looks very similar to what's in the paper but I can add the  website url if you want.